update-models.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
```

Comment:
I recommend adding something like `set -euo pipefail` so the script stops running if it hits problems